src/mlx90632.c

```diff
@@ -449,4 +471,43 @@ int32_t mlx90632_init(void)
 
     return 0;
 }
+
+int32_t mlx90632_start_measurement_burst(void)
+{
+    int32_t ret;
+    int tries = 150;
+    uint16_t reg;
+
+    ret = mlx90632_i2c_read(MLX90632_REG_CTRL, &reg);
+    if (ret < 0)
+        return ret;
+
+    reg |= MLX90632_CFG_SOB_MASK;
```

Comment:
Can we make a SOB_MASK  same macro as we have PWR_STATUS() stuff? Will make this more readable as it should actually produce something more descriptive than just operation with some mask